packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -98,50 +121,21 @@ const dynamicConfig = async (t, params) => {
  * @param {any} [params]
  */
 const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
-  const { zoe, installs, currency } = t.context;
-  const { terms, fakeInvitationPayment, registry } = await dynamicConfig(
-    t,
-    params,
-  );
-  const { timerService } = terms;
+  const { zoe, run: currency } = t.context;
   /** @type {MapStore<Brand, { setPrice: (r: Ratio) => void }>} */
   const priceAuthorities = makeScalarMapStore();
 
   // Each driver needs its own to avoid state pollution between tests
   const mockChainStorage = makeMockChainStorageRoot();
 
-  const pubsubTerms = harden({
-    storageNode: mockChainStorage.makeChildNode('thisAuction'),
-    marshaller: makeBoard().getReadonlyMarshaller(),
-  });
-
-  const { creatorFacet: GCF } = await E(zoe).startInstance(
-    installs.governor,
-    harden({}),
-    harden({
-      timer: timerService,
-      governedContractInstallation: installs.auctioneer,
-      governed: {
-        issuerKeywordRecord: {
-          Currency: currency.issuer,
-        },
-        terms: { ...terms, ...customTerms },
-        pubsubTerms,
-      },
-    }),
-    {
-      governed: {
-        initialPoserInvitation: fakeInvitationPayment,
-        ...pubsubTerms,
-      },
-    },
-  );
+  const { space, timer, registry } = await setupServices(t, params);
+  const { auctioneerKit: auctioneerKitP, reserveKit } = space.consume;
+  const auctioneerKit = await auctioneerKitP;
 
   /** @type {Promise<import('../../src/auction/auctioneer.js').AuctioneerPublicFacet>} */
   // @ts-expect-error xxx cast
-  const publicFacet = E(GCF).getPublicFacet();
-  /** @type {import('../../src/auction/auctioneer.js').AuctioneerLimitedCreatorFacet} */
-  const creatorFacet = E(GCF).getCreatorFacet();
+  const publicFacet = auctioneerKit.publicFacet;
+  const creatorFacet = auctioneerKit.creatorFacet;
```

Comment:
I think this the type (and cast) are wrong now. The type should flow from `auctioneerKit` in the space:
https://github.com/Agoric/agoric-sdk/blob/e7718c90cb5317b738aec636f568c4ebbdc4515f/packages/inter-protocol/src/proposals/econ-behaviors.js#L83

Please confirm then simplify these to:
```suggestion
  const {creatorFacet, publicFacet} = auctioneerKit;
```

Reply:
done